app/static/style.css

```diff
@@ -198,4 +198,113 @@ button {
 
 .error-group > p:last-child {
     padding-bottom: 9px;
-}
\ No newline at end of file
+}
+
+/*********************************/
+/* User Profile                        */
+/*********************************/
+
+
+
+img {
+    display: block;
+}
+
+.container {
+    max-width: 93.5rem;
+    margin: 0 auto;
+    padding: 0 2rem;
+}
+
+.btn {
+    display: inline-block;
+    font: inherit;
+    background: none;
+    border: none;
+    color: inherit;
+    padding: 0;
+    cursor: pointer;
+}
```

Comment:
It would be best to prefix these with profile- so that they dont effect the styling of other components, besides that looks good!